defaults/network-ents.go

```diff
@@ -11,10 +12,17 @@ const (
 )
 
 const (
-	NetworkNoneEntId  = "none"  // network.none
-	NetworkUserEntId  = "user"  // network.user
-	NetworkProxyEntId = "proxy" // network.proxy
-	NetworkAdminEntId = "admin" // network.admin
+	NetworkNoneEntFullId  = NetworkTLD + ".none"
```

Comment:
@n4ss: I suppose we could leave this as-is for easy referencing of a entitlement name that a user may want to add/remove/inspect, but I think that this library should do the parsing from configs to entitlement types for defaults that it knows about.

It might be helpful to have an example (and test!) showing how default entitlements are specified and ingested.  Perhaps I'm misunderstanding your desired flow here